packages/web-pkg/src/components/TextEditor/TextEditor.vue

```diff
@@ -1,25 +1,71 @@
 <template>
-  <app-loading-spinner v-if="loading" />
-  <div v-else id="text-editor-container" ref="toastUiEditorRef" :data-markdown-mode="isMarkdown" />
+  <div id="text-editor-container" class="oc-height-1-1">
+    <md-preview
+      v-if="isReadOnly"
+      id="text-editor-preview-component"
+      :model-value="currentContent"
+      no-mermaid
+      no-katex
+      no-highlight
```

Comment:
@kulmann we disabled code hilighting because the lib is around 1MB.

w/o it looks like this:

<img width="1491" alt="image" src="https://github.com/user-attachments/assets/0283d5ec-fbb4-4253-a735-ba72fc9c83b7" />

with:

<img width="1491" alt="image" src="https://github.com/user-attachments/assets/244edd3d-bde5-44fb-8ad9-05ee44637f0b" />


please make a decision ❤️